Fix broken documentation links

adrifoster opened the pull request:
### Description of changes

Fixes broken documentation links in tech note and users guide - WIP

### Specific notes

Some links I couldn't find - so we'll need to address those.

CTSM Issues Fixed (include github issue #):
- fixes #2767

Are answers expected to change (and if so in what way)? No

Any User Interface Changes (namelist or namelist defaults changes)? No

Does this create a need to change or add documentation? Did you do so? Yes

Commit:
links

Commit:
fixed more links

samsrabin commented on doc/source/users_guide/setting-up-and-running-a-case/customizing-the-clm-configuration.rst:
```diff
@@ -503,7 +503,7 @@ Like other datasets, if you want to use a given initial condition file to be use
 ------------------------------------
 
-For running "I" cases there are several other noteworthy configuration items that you may want to work with. Most of these involve settings for the DATM, but one ``CCSM_CO2_PPMV`` applies to all models. The list of DATM settings is `here <http://esmci.github.io/cime/data_models/data-atm.html>`_. If you are running a B, E, or F case that doesn't use the DATM obviously the DATM_* settings will not  ...
+For running "I" cases there are several other noteworthy configuration items that you may want to work with. Most of these involve settings for the DATM, but one ``CCSM_CO2_PPMV`` applies to all models. The list of DATM settings is (**link currently broken**). If you are running a B, E, or F case that doesn't use the DATM obviously the DATM_* settings will not be used. All of the settings below a ...
```
I've filed an issue about updating the CDEPS documentation (#3225). For now, just go ahead and link to the outdated page.

slevis-lmwg committed:
Add two 1x1 FatesColdPRT2 tests, one with, one without MIMICS

PASS ERS_D_Ld30.1x1_brazil.I2000Clm60FatesCrujraRs.derecho_intel.clm-FatesColdPRT2
FAIL ERS_D_Ld30.1x1_brazil.I2000Clm60FatesCrujraRs.derecho_intel.clm-FatesColdPRT2--clm-mimicsFatesCold--clm-nofireemis
The latter needs "nofireemis" to work with Fates, but it then dumps core in line 1180 SoilBiogeochemDecompCascadeMIMICSMod.F90

slevis-lmwg committed:
Uncomment if ( .not. use_fates_sp ) section of clmfates_interfaceMod

...calculating these variables:
nf_soil%decomp_npools_sourcesink_col
nf_soil%fates_litter_flux

slevis-lmwg committed:
Replace plant_ndemand(c) * nuptake_prof(c,j) with plant_ndemand_vr(c,j)

slevis-lmwg committed:
New use_fates section for plant_ndemand_vr and plant_ndemand from fates

slevis-lmwg committed:
New use_fates section to set the FATES N uptake fluxes

slevis-lmwg committed:
Change clm_fate from "in" to "inout" argument

slevis-lmwg committed:
Replace if (use_fates) with (is_fates(c)) to add flexibility

slevis-lmwg committed:
Remove two unhelpful 1x1 FatesColdPRT2 tests that I added earlier

slevis-lmwg committed:
New is_fates section for plant_ndemand, now when use_nitrif_denitrif

slevis-lmwg committed:
Add (really, uncomment) FATES npp to enable fixation

slevis-lmwg committed:
Uncomment hrv_deadstem_to_prod* lines in clmfates_interfaceMod

...for fates_parteh_mode == 2

slevis-lmwg committed:
Add error check about fates_parteh_mode = 2 and suplnitro = suplnAll

slevis-lmwg committed:
Upd. namelist_defaults for suplnitro when parteh_mode == 1 or /= 1

slevis-lmwg committed:
prt_carbon_allom_hyp -> fates_c_only & prt_cnp_flex_allom_hyp ->fates_cn

These variables originate in fates, so this renaming requires the same
renaming in fates; I will open the corresponding PR very soon

slevis-lmwg committed:
Correction to syntax error caught by gnu and nvhpc, not intel

slevis-lmwg committed:
Update fates paramfile var prescribed_puptake to 1 for PRT2 test

slevis-lmwg committed:
Adjust unit conversions to agree with FATES commit b2a1da6

slevis-lmwg committed:
Change fates paramfile var prescribed_puptake to 10 for PRT2 test

slevis-lmwg committed:
Replace fates_cn with fates_cnp throughout

slevis-lmwg committed:
Remove FATES_ERROR_EL from user_nl_clm (out-of-scope fix)

slevis-lmwg committed:
Merge tag 'ctsm5.4.004' into fates-cn

Update crop parameters and maturity requirements

slevis resolved conflicts:
src/fates <-- So far I have not changed .gitmodules to point to my branch
src/soilbiogeochem/SoilBiogeochemCompetitionMod.F90

slevis-lmwg committed:
Merge tag 'ctsm5.4.005' into fates-cn

FATES JSON parameter files

These changes accommodate a JSON format to the FATES parameter file.

Documentation has been updated in the FATES User's Guide:
https://fates-users-guide.readthedocs.io/en/latest/user/Parameter-File.html

fates was updated from sci.1.88.0_api.42.0.0 to sci.1.89.0_api.43.0.0

ESCOMP and NGEET PRs:
https://github.com/ESCOMP/ctsm/pull/3570
https://github.com/NGEET/fates/pull/1493

slevis resolved conflicts:
cime_config/testdefs/testmods_dirs/clm/FatesColdPRT2/shell_commands
cime_config/testdefs/testmods_dirs/clm/FatesColdPRT2/user_nl_clm
src/fates

slevis-lmwg committed:
Add PRT2 test with synthetic N to replicate the pre-CN-coupling setup

rgknox committed:
changing namelist type for fates_parteh_mode, part 1

slevis-lmwg committed:
Update two namelist error-checks

slevis-lmwg committed:
Update equivalent to last error-checks, this time in controlMod.F90

slevis-lmwg committed:
Add FatesColdPRT2_suplnAll test and corresponding testmod

rgknox committed:
more name changes for fates CNP and c-only

rgknox committed:
working on passing nl defaults

slevis-lmwg committed:
Merge branch 'fates-cn' into fates-cn

rgknox committed:
fixes to namelist processing of suplnitro

rgknox committed:
more explicit logic on suplnitro with fates

rgknox committed:
more explicit namelist defaults

slevis-lmwg committed:
Remove trailing whitespace

slevis-lmwg committed:
Changes for namelist updates to work

slevis-lmwg committed:
Merge pull request #19 from rgknox/fates-cn

name changes (WIP)

rgknox committed:
Merge tag 'ctsm5.4.019' into fates-cn-merge

Fix years used by generate_gdds

rgknox committed:
merge with  5.3.026

rgknox committed:
Added uptake fluxes to restarts for fates, pass in boundary condition structure

slevis-lmwg committed:
Merge pull request #20 from rgknox/fates-cn

restarting uptake fluxes

slevis-lmwg committed:
Merge branch 'master' into fates-cn

slevis-lmwg committed:
Simplify elsif in CLMBuildNamelist.pm to get past failure

slevis-lmwg committed:
Correct minor typo

slevis-lmwg committed:
Add 2 expected failure tests to build-namelist_test.pl

slevis-lmwg committed:
Merge tag 'ctsm5.4.032' into fates-cn

Simplify messaging during docs build

slevis resolved conflicts:
bld/namelist_files/namelist_defaults_ctsm.xml

slevis-lmwg committed:
Update to existing compsets in new PRT2 tests for cases to create

mvdebolskiy committed:
move getandsettime call under read condition in clmfates%restart

samsrabin commented:
It looks like there should be a built-in Sphinx link checker! https://sphinx-intro-tutorial.readthedocs.io/en/latest/sphinx_extensions.html#linkcheck

- [x] **Not in this PR** Maybe not this PR; if skipping, add an issue about putting this in our doc build scripts / CI.

slevis-lmwg committed:
Merge tag 'ctsm5.4.034' into fates-cn

bug fix to the FATES land use driver input code

This PR includes a cherry-picked fix from NorESMhub/CTSM#209 to address swap in the
order of rangeland and pasture in the reading of the landuse drivers

slevis resolved conflicts:
src/fates

slevis-lmwg committed:
Remove passing test from expected failures

slevis-lmwg committed:
Update .gitmodules with the new FATES tag (ie, FATES-side fates-cn)

slevis-lmwg committed:
Remove machine lawrencium from testlist_clm.xml

slevis-lmwg committed:
Add comments about suplnitro in namelist_defaults_ctsm.xml

slevis-lmwg committed:
Change fates_parteh_mode to SHR_KIND_CS (char*80 inst. of 256)

slevis-lmwg committed:
Add comments clarifying the presence of two very similar loops

slevis-lmwg committed:
Merge remote-tracking branch 'escomp/master' into fates-cn

slevis-lmwg committed:
Update total number of tests in build-namelist_test.pl

slevis-lmwg committed:
Draft updates to ChangeLog/ChangeSum

slevis-lmwg committed:
Revert 25ea275 by putting FATES_ERROR_EL back in user_nl_clm

slevis-lmwg committed:
Merge pull request #3409 from slevis-lmwg/fates-cn

ctsm5.4.036: Complete the FATES-CLM nitrogen coupling

glemieux committed:
Merge tag 'ctsm5.4.036' into mirror-noresm-fatestimer

Complete the FATES-CLM nitrogen coupling

PR #3409. Description:
Intoduce code, namelist option, and tests on the CLM side to accommodate interactive nitrogen with FATES.

Roundoff changes for most cases, though not for Sp and FatesSp.

The FATES side PR is https://github.com/NGEET/fates/pull/1472. Fates got updated from sci.1.92.1_api.44.1.0 to sci.1.92.4_api.45.0.0.

Supporting information (other than in the issue referenced below):
- Nutrient enabled FATES handbook: https://docs.google.com/document/d/1I35fGDfKTkn9_8Z6qXot7HZf3ICSpLONd-iFEErAh0k/edit?usp=drive_link
- FATES CLM N coupling: https://docs.google.com/document/d/1mpBtpCLGJpAGw6R3-nGGY92IWRU3ISvaAUKxvcDVJRw/edit?usp=drive_link

Resolves issue #3378.

samsrabin committed:
Update ChangeLog/Sum.

samsrabin committed:
Merge pull request #3940 from rgknox/mirror-noresm-fatestimer

ctsm5.4.037: mirror: ctsm5.4.002_noresm_v6: fix for (fates) year-boundary restart issue

slevis-lmwg committed:
Merge remote-tracking branch 'escomp/master' into merge-b4bdev-20260507

Merges ctsm5.4.037 to b4b-dev

slevis-lmwg committed:
Merge remote-tracking branch 'escomp/b4b-dev' into merge-b4bdev-20260507

Bug-fix provided by Sam R. to avoid the "check docs against style guide" error in github's workflows

slevis-lmwg committed:
Draft ChangeLog/Sum

slevis-lmwg committed:
Merge pull request #4000 from slevis-lmwg/merge-b4bdev-20260507

Merge b4bdev 20260507

Commit:
merge in master

Commit:
update testing

Commit:
easy fixes

Commit:
more fixes

Commit:
more links

Commit:
even more links

Commit:
Merge branch 'b4b-dev' into fix-docs-links

Commit:
Merge branch 'b4b-dev' into fix-docs-links

samsrabin committed:
Point to outdated CDEPS DATM documentation.

I've included a disclaimer about that page being out of date. I've also mentioned that this link should be updated once the CDEPS page is; see https://github.com/ESCOMP/CTSM/issues/3225.
